hangups/ui/__main__.py

```diff
@@ -31,18 +33,21 @@
         ('inactive_tab', 'underline', 'light green'),
         ('msg_date', 'dark cyan', ''),
         ('msg_sender', 'dark blue', ''),
+        ('msg_self', 'light blue', ''),
```

Comment:
Did `light blue` work for you? It's the same default colour as the message text for me. Strangely, it looks correct in urwid's `palette_test.py`.